event_registration_analytic/security/event_registration_analytic.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="event_track_assistant.event_rule_personal_session" model="ir.rule">
+            <field name="domain_force">[('session_teacher_ids','in',user.partner_id.id)]</field>
```

Comment:
¿Esto funciona? Ya que la condición en principio es al reves, user.partner_id.id in session_teacher_ids si en session_teacher_ids tienes varios, y el partner del user está más o menos en medio, ¿funciona?